mk_verificator/tests/packet_checker/test_packet_checker.py

```diff
@@ -4,13 +4,15 @@
 
 
 @pytest.mark.parametrize(
-    ("group"),
+    "group",
     utils.get_groups(utils.get_configuration(__file__))
 )
 def test_check_package_versions(local_salt_client, group):
+    config = utils.get_configuration(__file__)
+
     output = local_salt_client.cmd(group, 'lowpkg.list_pkgs')
 
-    if len(output.keys()) < 2:
+    if len(output.keys()) < config["min_nodes"]:
```

Comment:
Why do we need this? This is pretty straight-forward - if we have less than 2 nodes - do not check results. Also we can rephrase like "if it is only 1 node of a kind".